source/user-manual/capabilities/system-inventory/compatibility-matrix.rst

```diff
@@ -25,3 +25,27 @@ The Syscollector module supports different options across various operating syst
 +------------------------+-----------+-----------+-----------+----------+-----------+-----------+-----------+
 |    Solaris             |     ✗     |     √     |     √     |     √    |     ✗     |     ✗     |     –     |
 +------------------------+-----------+-----------+-----------+----------+-----------+-----------+-----------+
+
+.. versionadded:: 4.7.0
+
+The Syscollector module now supports scan packages from `NPM (Node Package Manager)  <https://https://www.npmjs.com/>`_, `PyPI (Python Package Index)  <https://https://www.npmjs.com/>`_ and `Snap store <https://snapcraft.io/>`_ , **the support is only for the default paths of those packages' installation**.
+
+The following table shows the new feature that are compatible with various operating systems, where '*√*' means compatible enhancement, '*✗*' not supported enhancement, and '*–*' not applicable.
```

Comment:
I wouldn't repeat this definition, but let the team decide, it is just an opinion

Reply:
I think the same, just once seems enough.

Reply:
I see the PR #15740 has for target `master`, I agree, I'll remove and let _Snap_ to future releases.

Reply:
Thank you! Snap will have to wait.
But we still have the duplicate note.